src/styles/components/_venue.scss

```diff
@@ -93,12 +94,11 @@
     }
 
     &_description {
-        padding: 60px;
         background-color: $color-provincial-pink;
         padding: 60px;
```

Comment:
I still think the padding on the sides should be less :) When the padding: 60px; it means all the sides are set to be 60px, we can reduce the right and left in half perhaps? padding: 60px 30px;

Reply:
pushed changes